optimum/amd/brevitas/export.py

```diff
@@ -0,0 +1,10 @@
+import torch
+from brevitas.export.onnx.standard.qcdq.manager import StdQCDQONNXManager
+from brevitas_examples.llm.llm_quant.export import brevitas_proxy_export_mode
+
+from optimum.exporters.onnx import onnx_export_from_model
+
+
+def export_quantized_model(quantized_model, path, task="text-generation-with-past"):
```

Comment:
For this function lets match the necessary input signatures from here:  https://github.com/huggingface/optimum/blob/main/optimum/exporters/onnx/convert.py#L906

Reply:
Some of the arguments for the export function have very specific values at the moment (e.g., `task` or `do_validation`, etc.), that are needed for the export to work as expected.
Should those be matched as well or left to their current values without possibility of being changed?

Reply:
We should add only the required values for this context. and set default for `do_validation` to `false` and task to `text-generation-with-past`